.github/workflows/on-safe-to-test-label.yml

```diff
@@ -69,24 +69,31 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Install Development Tools
+        run: |
+          sudo yum groupinstall "Development Tools" -y
       - name: Install Go
         run: |
           mkdir /home/ec2-user/.cache
           mkdir /home/ec2-user/.cache/go-build
           mkdir /home/ec2-user/.cache/go-mod
           mkdir /home/ec2-user/go
-          sudo yum install golang -y
+          GOVERSION=go1.17.7
+          wget https://go.dev/dl/$GOVERSION.linux-amd64.tar.gz
+          sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf $GOVERSION.linux-amd64.tar.gz
+          PATH="$PATH:/usr/local/go/bin"
```

Comment:
Is there a way to specify version through `yum` instead of hard coding the url?

Reply:
Yum only has up to version 1.15